src/components/Onfido/index.native.js

```diff
@@ -39,30 +40,53 @@ function Onfido({sdkToken, onUserExit, onSuccess, onError}) {
                     return;
                 }
 
-                // Handle user camera permission on iOS and Android
-                if (_.contains([CONST.ONFIDO.ERROR.USER_CAMERA_PERMISSION, CONST.ONFIDO.ERROR.USER_CAMERA_DENINED, CONST.ONFIDO.ERROR.USER_CAMERA_CONSENT_DENIED], errorMessage)) {
-                    Alert.alert(
-                        translate('onfidoStep.cameraPermissionsNotGranted'),
-                        translate('onfidoStep.cameraRequestMessage'),
-                        [
-                            {
-                                text: translate('common.cancel'),
-                                onPress: () => onUserExit(),
-                            },
-                            {
-                                text: translate('common.settings'),
-                                onPress: () => {
-                                    onUserExit();
-                                    Linking.openSettings();
-                                },
-                            },
-                        ],
-                        {cancelable: false},
-                    );
-                    return;
-                }
+                if (!_.isEmpty(errorMessage)) {
+                    const micPermission = Platform.select({
```

Comment:
`Platform.select` is not preferred. Please re-use `iou/ReceiptSelector/CameraPermission` structure to handle platform specific permissions.